library/src/main/kotlin/it/krzeminski/githubactions/actions/MissingAction.kt

```diff
@@ -0,0 +1,14 @@
+package it.krzeminski.githubactions.actions
+
+/**
+ * Used by the script generator when no Action Wrapper can be found
+ */
+class MissingAction(
+    override val actionOwner: String,
+    override val actionName: String,
+    override val actionVersion: String,
+    val with: Map<String, String>,
```

Comment:
```suggestion
    val inputs: Map<String, String>,
```
? `with` suggests that also non-input parameters can be passed here. I know `wirh` is in line with GitHub API, however we're going to have `_customInputs` or `_freeInputs` in all generated wrappers, where `_customWith` would look weird :smile: